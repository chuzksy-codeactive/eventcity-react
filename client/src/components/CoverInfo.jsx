import React, { Component } from 'react';
import { Link } from 'react-router-dom';

class CoverInfo extends Component {
  render() {
    return (
      <div className="info">
        <div>
          <h1>Welcome to EventCity!</h1>
        </div>
        <div>
          <p>
            At EventCity! we pride ourselves on the unrivalled personal {`event`} services, we provide to our clientele. We guide you from the
            stressful decision making {`process`}, ensuring you are comfortable, whether it is a wedding, corporate {`function `}or even a kiddies
            party, we create a buzz around you, taking you to the next level.
          </p>
        </div>
        <div>
          <h3>Innovation, {`Performance`} and Delivery</h3>
        </div>
        <Link to="/login">
          <button type="button" className="btn btn-success btn-lg">
            Sign In here
          </button>
        </Link>
      </div>
    );
  }
}

export default CoverInfo;
