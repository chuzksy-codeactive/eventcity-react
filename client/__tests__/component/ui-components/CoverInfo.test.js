import React from 'react'
import CoverInfo from '../../../src/components/ui-components/CoverInfo'

describe('CoverInfo Component test', () => {
  it('It should render the component without error', () => {
    const wrapper = shallow(<CoverInfo />);
    expect(wrapper).toMatchSnapshot();
  });
})