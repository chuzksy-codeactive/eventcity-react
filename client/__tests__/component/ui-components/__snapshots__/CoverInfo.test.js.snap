// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoverInfo Component test It should render the component without error 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CoverInfo />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div>
          <h1>
            Welcome to EventCity!
          </h1>
        </div>,
        <div>
          <p>
            At EventCity! we pride ourselves on the unrivalled personal 
            event
             services, we provide to our clientele. We guide you from the stressful decision making 
            process
            , ensuring you are comfortable, whether it is a wedding, corporate 
            function 
            or even a kiddies party, we create a buzz around you, taking you to the next level.
          </p>
        </div>,
        <div>
          <h3>
            Innovation, 
            Performance
             and Delivery
          </h3>
        </div>,
        <Link
          id="login"
          replace={false}
          to="/login"
        >
          <button
            className="btn btn-success btn-lg"
            type="button"
          >
            Sign In here
          </button>
        </Link>,
      ],
      "className": "info",
      "id": "sign-info",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h1>
            Welcome to EventCity!
          </h1>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Welcome to EventCity!",
          },
          "ref": null,
          "rendered": "Welcome to EventCity!",
          "type": "h1",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <p>
            At EventCity! we pride ourselves on the unrivalled personal 
            event
             services, we provide to our clientele. We guide you from the stressful decision making 
            process
            , ensuring you are comfortable, whether it is a wedding, corporate 
            function 
            or even a kiddies party, we create a buzz around you, taking you to the next level.
          </p>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "At EventCity! we pride ourselves on the unrivalled personal ",
              "event",
              " services, we provide to our clientele. We guide you from the stressful decision making ",
              "process",
              ", ensuring you are comfortable, whether it is a wedding, corporate ",
              "function ",
              "or even a kiddies party, we create a buzz around you, taking you to the next level.",
            ],
          },
          "ref": null,
          "rendered": Array [
            "At EventCity! we pride ourselves on the unrivalled personal ",
            "event",
            " services, we provide to our clientele. We guide you from the stressful decision making ",
            "process",
            ", ensuring you are comfortable, whether it is a wedding, corporate ",
            "function ",
            "or even a kiddies party, we create a buzz around you, taking you to the next level.",
          ],
          "type": "p",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h3>
            Innovation, 
            Performance
             and Delivery
          </h3>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Innovation, ",
              "Performance",
              " and Delivery",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Innovation, ",
            "Performance",
            " and Delivery",
          ],
          "type": "h3",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <button
            className="btn btn-success btn-lg"
            type="button"
          >
            Sign In here
          </button>,
          "id": "login",
          "replace": false,
          "to": "/login",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Sign In here",
            "className": "btn btn-success btn-lg",
            "type": "button",
          },
          "ref": null,
          "rendered": "Sign In here",
          "type": "button",
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div>
            <h1>
              Welcome to EventCity!
            </h1>
          </div>,
          <div>
            <p>
              At EventCity! we pride ourselves on the unrivalled personal 
              event
               services, we provide to our clientele. We guide you from the stressful decision making 
              process
              , ensuring you are comfortable, whether it is a wedding, corporate 
              function 
              or even a kiddies party, we create a buzz around you, taking you to the next level.
            </p>
          </div>,
          <div>
            <h3>
              Innovation, 
              Performance
               and Delivery
            </h3>
          </div>,
          <Link
            id="login"
            replace={false}
            to="/login"
          >
            <button
              className="btn btn-success btn-lg"
              type="button"
            >
              Sign In here
            </button>
          </Link>,
        ],
        "className": "info",
        "id": "sign-info",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h1>
              Welcome to EventCity!
            </h1>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Welcome to EventCity!",
            },
            "ref": null,
            "rendered": "Welcome to EventCity!",
            "type": "h1",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <p>
              At EventCity! we pride ourselves on the unrivalled personal 
              event
               services, we provide to our clientele. We guide you from the stressful decision making 
              process
              , ensuring you are comfortable, whether it is a wedding, corporate 
              function 
              or even a kiddies party, we create a buzz around you, taking you to the next level.
            </p>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "At EventCity! we pride ourselves on the unrivalled personal ",
                "event",
                " services, we provide to our clientele. We guide you from the stressful decision making ",
                "process",
                ", ensuring you are comfortable, whether it is a wedding, corporate ",
                "function ",
                "or even a kiddies party, we create a buzz around you, taking you to the next level.",
              ],
            },
            "ref": null,
            "rendered": Array [
              "At EventCity! we pride ourselves on the unrivalled personal ",
              "event",
              " services, we provide to our clientele. We guide you from the stressful decision making ",
              "process",
              ", ensuring you are comfortable, whether it is a wedding, corporate ",
              "function ",
              "or even a kiddies party, we create a buzz around you, taking you to the next level.",
            ],
            "type": "p",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h3>
              Innovation, 
              Performance
               and Delivery
            </h3>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Innovation, ",
                "Performance",
                " and Delivery",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Innovation, ",
              "Performance",
              " and Delivery",
            ],
            "type": "h3",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <button
              className="btn btn-success btn-lg"
              type="button"
            >
              Sign In here
            </button>,
            "id": "login",
            "replace": false,
            "to": "/login",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Sign In here",
              "className": "btn btn-success btn-lg",
              "type": "button",
            },
            "ref": null,
            "rendered": "Sign In here",
            "type": "button",
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
